website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -2,79 +2,95 @@
 
+</TabItem>
+<TabItem label="AWS EKS" value="AWSEKS">
+
+   1. Set up the Kubernetes cluster by [creating a kubeconfig for Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html).
+    2. Ensure that you have a default storage class running. If not, [create default storage class](https://docs.aws.amazon.com/eks/latest/userguide/storage-classes.html)
+    3. The NGINX Ingress Controller should be enabled by default. If not, [install NGINX Controller for AWS EKS](https://kubernetes.github.io/ingress-nginx/deploy/#network-load-balancer-nlb).
+    Ensure that you enable the correct version for your cluster.
+
+</TabItem>
+<TabItem label="Microk8s" value="Microk8s">
+
+  1. Set up Kubernetes by following [working with kubectl](https://microk8s.io/docs/working-with-kubectl).
+  2. Ensure that you have a default storage class running. If not, [enable storage class](https://microk8s.io/docs/command-reference#heading--microk8s-enable).
+  3. The NGINX Ingress Controller should be enabled by default. If not, [add on: Ingress](https://microk8s.io/docs/addon-ingress). Ensure that you enable the correct version for your cluster.
+    
+</TabItem>
+</Tabs>
+
+## Install
+
+1. Click to download the [`values.yaml`](https://bit.ly/3ETEgPT) file, and place it into the Appsmith installation folder.
+
+OR, run the following cURL:
+
+```bash
+curl -L https://bit.ly/3ETEgPT -o $PWD/values.yml
 ```
+
+2. Add Appsmith into your repository using Helm.
+
+```bash
 helm repo add appsmith https://helm.appsmith.com
 
 helm repo update
 ```
 
-2\. Install the chart with the <mark>`appsmith`</mark> release.
+2. Install the chart with the <mark>`appsmith`</mark> release.
 
-```
+```bash
 helm install appsmith/appsmith --generate-name
 ```
 
-The command deploys the Appsmith application on the Kubernetes cluster in the default configuration. The [Parameters](https://github.com/appsmithorg/appsmith/tree/release/deploy/helm#paramters) section lists the configurable parameters during installation.
+The command deploys Appsmith on the Kubernetes cluster with the default configuration. Read more on [configurations](#configuration) that you could do during installation.
 
-### Uninstalling the Chart
 
-To uninstall the <mark>`appsmith`</mark> release:
-
-```
-helm list
-NAME                       NAMESPACE       REVISION        UPDATED                                 STATUS          CHART           APP VERSION
-appsmith-1631069261        default         1               2021-09-09 11:24:40.152766 +0700 +07    deployed        appsmith-1.3.0  1.16.0
-
-helm uninstall appsmith-1631069261
-```
-
-The command uninstalls the release and removes all Kubernetes resources associated with the chart.
+## Configuration
```

Comment:
Let's stay consistent with the headings. In the above section, it starts with 'Install', so here it should be 'Configure the Helm chart'

Reply:
I can reword it to configure